stock_quant_manual_assign/__openerp__.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Mikel Arregi - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "Stock - Manual assignment of quants",
+    "version": "8.0.1.0.0",
```

Comment:
@pedrobaeza Sorry, my PR message is not very clear. I'm just adding this V8 module as unported  in V9 (i will create similar PR for stock_quant_merge and partner_location_auto_create).

We are merging, with @gurneyalex, 8.0 into 9.0 (splitted by module).